macros/on-run-end/print_dbt_project_evaluator_issues.sql

```diff
@@ -33,7 +33,7 @@
         {% set db_schema = model_database ~ "." ~ model_schema if model_database else model_schema %}
 
         {% set sql_statement %}
-        select * from {{db_schema}}.{{name_model_checked}}
+        select * from {{ model_details['relation_name'] }}
```

Comment:
I think that we can use the `.` notation here and remove the variables from above as they are no longer needed.

```suggestion
        select * from {{ model_details.relation_name }}
```

Reply:
Yes, you're absolutely right. Fixed that.